.travis.yml

```diff
@@ -0,0 +1,14 @@
+language: python
+
+sudo: false
+
+python:
+  - "3.5"
```

Comment:
We do not need to support `python3.5`. This app is a standalone service.
So, only `python3.6` is good enough.